debian/marquee/scripts/fdupes

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+# This need https://github.com/tobiasschulz/fdupes for --linkhard support.
+fdupes --recurse --linkhard --noprompt /var/www/encoding.spec.whatwg.org
```

Comment:
Why not apply this everywhere?

Reply:
It takes longer to run the more files there are, and we wouldn't gain much by it:
```
$ fdupes --recurse --noempty --summarize /var/www/
303 duplicate files (in 35 sets), occupying 4.8 megabytes
```

That's quite meager compared to the gigabytes saved for encoding. And, there's always the chance that there are fdupes bugs or yet-unknown downsides to hardlinking, so I'm being conservative.